modules/claims_api/app/sidekiq/claims_api/report_hourly_unsuccessful_submissions.rb

```diff
@@ -4,6 +4,11 @@ module ClaimsApi
   class ReportHourlyUnsuccessfulSubmissions < ClaimsApi::ServiceBase
     sidekiq_options retry: 7
 
+    NO_INVESTIGATION_ERROR_TEXT = [
+      'The Maximum number of EP codes have been reached for this benefit type claim code',
+      'Claim could not be established. Retries will fail.'
+    ].freeze
+
```

Comment:
This is the test in the `evss_response` for these errors.  I tried to grab the most concise, but also the most obvious chunk of text for each one.